docs/user/security/securing-communications/elasticsearch-mutual-tls.asciidoc

```diff
@@ -5,36 +5,45 @@
-authenticated. Most requests made by end users through {kib} to {es} are authenticated by using the credentials of the logged-in user. There
-are, however, a few internal requests that {kib} needs to make to {es}. For this reason, you must configure credentials for {kib} to use for
-those requests.
-
-If {kib} has `elasticsearch.username` and `elasticsearch.password` configured, it will attempt to use these to authenticate to {es} via the
-{ref}/native-realm.html[native realm]. However, {kib} also supports mutual TLS authentication with {es} via a {ref}/pki-realm.html[Public
-Key Infrastructure (PKI) realm]. To do so, {es} needs to verify the signature on the {kib} client certificate, and it also needs to map the
-client certificate's distinguished name (DN) to the appropriate `kibana_system` role.
-
-NOTE: Using a PKI realm is a gold feature. For a comparison of the Elastic license levels, see https://www.elastic.co/subscriptions[the
-subscription page].
-
-To configure {kib} and {es} to use mutual TLS authentication:
-
-. <<using-kibana-with-security,Set up {kib} to work with {stack} {security-features} with a username and password>>.
-
-. <<configuring-tls-kib-es,Set up TLS encryption between {kib} and {es}>>.
-+
-This entails generating a "server certificate" for {es} to use on the HTTP layer.
+Secure Sockets Layer (SSL) and Transport Layer Security (TLS) provide encryption
+for data-in-transit. While these terms are often used interchangeably, {kib}
+supports only TLS, which supersedes the old SSL protocols.
+
+TLS requires X.509 certificates to authenticate the communicating parties and
+perform encryption of data-in-transit. Each certificate contains a public key
+and has and an associated -- but separate -- private key; these keys are used
+for cryptographic operations. {kib} supports certificates and private keys in
+PEM or PKCS#12 format.
+
+In a standard TLS configuration, the server presents a signed certificate to
+authenticate itself to the client. In a mutual TLS configuration, the client
+also presents a signed certificate to authenticate itself to the server.
+
+{es} {security-features} are enabled on your cluster by default, so each request
+that {kib} (the client) makes to {es} (the server) is authenticated. Most
+requests made by end users through {kib} to {es} are authenticated by using the
+credentials of the logged-in user.
+
+If {kib} has `elasticsearch.username` and `elasticsearch.password` configured,
+it attempts to use these credentials to authenticate to {es} via the
+{ref}/native-realm.html[native realm]. However, {kib} also supports mutual TLS
+authentication with {es} via a
+{ref}/pki-realm.html[Public Key Infrastructure (PKI) realm]. With this setup,
+{es} needs to verify the signature on the {kib} client certificate, and it also
+needs to map the client certificate's distinguished name (DN) to the appropriate
+`kibana_system` role.
```

Comment:
The section above states "Elasticsearch security features are enabled on your cluster by default".
However, when setting up Kibana with the enrollment token, it doesn't use username/password to authenticate to ES, it just uses the service account token.

Perhaps we should change this sentence to reflect that?

Reply:
@jportner, solid point. Does this change reflect the updated behavior?
>When you enroll {kib} with an {es} cluster, the generated enrollment token is used to authenticate with {es}. {kib} also supports mutual TLS...